scripts/Get-Padded-Version.ps1

```diff
@@ -0,0 +1,79 @@
+<#
+.SYNOPSIS
+  Pad the revision number if the passed version is using a 4-part version scheme.
+
+.DESCRIPTION
+  Adds padded zeroes to the revision part of the version if the passed
+  Version parameter is using a 4-part scheme and the Parameter OnlyBelowVersion
+  is either $null or greater than the Version parameter.
+
+.PARAMETER Version
+  The version number to add the revision padding to (Pre-Releases is not supported).
+
+.PARAMETER OnlyBelowVersion
+  Only add padded zeroes to a Version when it is below this parameter, or always
+  add padded zeroes if this parameter is $null.
+
+.PARAMETER RevisionLength
+  The total length of the revision number part
+
+.OUTPUTS
+  The full padded version, or the specified Version if padding was not needed.
+
+.EXAMPLE
+  Get-Padded-Version -Version '24.0.0.195'
+  will output '24.0.0.195000' if the nuspec version is less than the version
+
+.EXAMPLE
+  Get-Padded-Version -Version '24.0.0.195'
+  will output '24.0.0.195001' if the nuspec version is equal to '24.0.0.195'
+  or equal to '24.0.0.195000'
+
+.EXAMPLE
+  Get-Padded-Version -Version '24.0.1.2' -OnlyBelowVersion '24.0.1'
+  Will output the same version that was given without padded zeroes.
+#>
+
+function Get-Padded-Version() {
+  param(
+    [Parameter(Mandatory = $true, Position = 0)]
+    [string]$Version,
+    [Parameter(Position = 1)]
+    [version]$OnlyBelowVersion = $null,
+    [Parameter(Position = 2)]
+    [int]$RevisionLength = 6
```

Comment:
No particular reason, I basically picked a random number between 4 and 10.
6 Seemed like a good fit though, IMO